### PR TITLE
Filling speakers

### PR DESCRIPTION
**Pull request by grechut**

### CODE REVIEW

File: speakers/list.json [hunk 1]
```diff
@@ -78,13 +271,44 @@
...
+        "date": "2018-07-18",
+        "title": "Bezbłędny frontend z Elm",
+        "urls": {
+          "slides": "http://bit.ly/2v2z5tS"
+        }
+      },
+      {
+        "meetup": 21,
+        "date": "2016-09-28",
+        "title": "o SOLIDnej architekturze aplikacji webowych",
```
Comment (jsynowiec) [inline speakers/list.json]: Typo? Should start with uppercase?

**Review by jsynowiec — COMMENTED** [review]
Found one potential typo. Otherwise utACK 👍 Did you check if it builds and how it looks?

**Comment by grechut** [comment]
image

We are not displaying meetup number when `post_link` cannot link to valid point.
I tried to quickly change `{% post_link meetjs-katowice-<%= talk.meetup %> #<%= talk.meetup %> %}` to handle that, but not successfully. Any idea on how to display text in case of missing post?

**Comment by jsynowiec** [comment]
Let's leave this as is for now. I'll patch the templates later.

**Pull request merged**